### ISSUE TITLE
Produce ROC and PR curves for both train and test sets

### ISSUE THREAD

**Issue by erikr** [broadinstitute/ml4h#332]
**What**
Produce ROC and PR curves for both train and test sets

**Why**
So we can assess overfitting.

**How**
Currently we produce loss curves for train and validation, and ROC curves for test.

However it can be unclear exactly where on the loss curve we stop at, and apply to test.

It would be helpful to also produce ROC and PR curves for both train and test sets.

From @paolodi:
> This line makes the test set plots: https://github.com/broadinstitute/ml/blob/c592c1b0a39ef7f4a27c0241ed42e85fe50aa3d4/ml4cvd/recipes.py#L146
> and one could replicate it to execute it also on the training set

**Acceptance Criteria**
`train` has an option to produce ROC and PR curves for the train set too.
